team-bayou/UNO-Shareboard
# Enhancement/#109/user flag

To Test:
-Add a user via postman
-Do a Get user by id to get that user and see if the conversion between int flag and boolean works
